modules/master-node-sizing.adoc

```diff
@@ -109,7 +109,13 @@ Operator Lifecycle Manager (OLM ) runs on the control plane nodes and it's memor
 
 [IMPORTANT]
 ====
-If you used an installer-provisioned infrastructure installation method, you cannot modify the control plane node size in a running {product-title} {product-version} cluster. Instead, you must estimate your total node count and use the suggested control plane node size during installation.
+You can modify the control plane node size in a running {product-title} {product-version} cluster for the following configurations only:
+
+* Clusters installed with a user-provisioned installation method.
+* AWS clusters installed with an installer-provisioned infrastructure installation method.
+* Clusters that use a control plane machine set to manage control plane machines.
```

Comment:
Thinking about how I did the control plane vs compute machine set docs updates for 4.12, I think this bullet should probably actually not go in anything prior to 4.12 (because I did not backport my distinction to anything earlier than that).